ingestion_tools/dataset_configs/gjensen/10162.yaml

```diff
@@ -18,7 +18,7 @@ datasets:
       id: null
       name: ATCC 15444
     cross_references:
-      publications: https://doi.org/10.1111/mmi.13593
+      publications: 10.1111/mmi.13593
```

Comment:
These https://doi.org/ prefixes are replaced with a re-run of the proposed new gjensen_config.py changes